settings/options.html

```diff
@@ -1,15 +1,14 @@
 <!DOCTYPE html>
-  <head>
-    <meta charset="utf-8">
-    <link rel="stylesheet" href="options.css"/>
-  </head>
-  <body>
-    <form>
-      <label>Mario mode<input type="checkbox" id="mario"></label>
-      <br><button type="submit">Save</button>
-    </form>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="options.css"/>
+</head>
+<body>
+  <form>
+    <label>Mario mode<input type="checkbox" id="mario"></label>
```

Comment:
Okay, I did some testing, and have some more comments…  🙂
1) We should really default this to "checked", so that people can have all the whimsy right off the bat.